### PR TITLE
Fix channels label size and alignment

### PR DESCRIPTION
**Pull request by kulczy**
| Q               | A
| --------------- | -----
| Branch?         | 1.4
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

channel

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Common/_channel.html.twig [hunk 1]
```diff
@@ -1 +1,17 @@
...
+    <div class="channel">
+        <span class="ui medium empty circular label channel__item" style="background-color: {{ channel.color|default('#000') }}"></span>
+
+        {% if channel.name is defined %}
+            <span class="channel__item">
+                {{ channel.name }}
+            </span>
+        {% endif %}
+
+        {% if channel.code is defined %}
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Common/_channel.html.twig]: What would you say about the unification of this logic, to always present a name and label of the channel? 

Also, then we can just pass the whole object to this template.

File: src/Sylius/Bundle/AdminBundle/Resources/views/Common/_channel.html.twig [hunk 2]
```diff
@@ -1 +1,12 @@
...
+{% if channel is defined %}
+    <div class="channel">
+        <span class="ui medium empty circular label channel__item" style="background-color: {{ channel.color|default('#000') }}"></span>
+
+        {% if channel.name is defined %}
+            <span class="channel__item">
+                {{ channel.name }}
+            </span>
+        {% endif %}
+
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Common/_channel.html.twig]: Unnecessary blank line

File: src/Sylius/Bundle/AdminBundle/Resources/views/Common/_channel.html.twig [hunk 3]
```diff
@@ -1 +1,12 @@
-<span class="ui small empty circular label" style="background-color: {{ channel.color }}"></span> {{ channel.name }} ({{ channel.code }})
+{% if channel is defined %}
+    <div class="channel">
+        <span class="ui medium empty circular label channel__item" style="background-color: {{ channel.color|default('#000') }}"></span>
+
+        {% if channel.name is defined %}
+            <span class="channel__item">
+                {{ channel.name }}
+            </span>
+        {% endif %}
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Common/_channel.html.twig]: Let's remove this if and write a default value
```suggestion
        <span class="channel__item">
            {{ channel.name|default(channel.name) }}
        </span>
```

**Comment by GSadee** [comment]
Thanks, Szymon! :1st_place_medal:

**Pull request merged**